src/renderer/library/components/publication/PublicationListElement.tsx

```diff
@@ -70,22 +70,38 @@ export class PublicationListElement extends React.Component<IProps, IState> {
     }
 
     public render(): React.ReactElement<{}> {
+
         const pub = this.props.publicationViewMaybeOpds;
-        const publishers = pub.publishers as Array<string | IOpdsContributorView>;
-        const formatedPublishers = publishers
-            .reduce(
-                (pv, cv) => {
-                    if ((cv as IOpdsContributorView)?.name) {
-                        return [...pv, `${pv}${(cv as IOpdsContributorView).name}`];
-                    }
-                    return cv && typeof cv === "string" ? [...pv, cv] : pv;
-                }, [])
-            .join(", ");
-        let formatedPublishedYear = "";
+        const publishers = pub.publishers;
+        const publishedAt = pub.publishedAt;
+        const isPublishers = !!publishers || !!publishedAt;
```

Comment:
if `pub.publishers` is not an array (e.g. undefined/null), and `pub.publishedAt` is truthy, then the line below crashes: `for (const publisher of publishers)`, right?

Reply:
`TypeError: v is not iterable`